### PR TITLE
Add auth for people api

### PR DESCRIPTION
**Pull request by DariuszMusielak**
Added user authentication for People Api. Save it in cookie and reuse it every request.

### CODE REVIEW

File: src/components/users/LoginPage.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        <form onSubmit={this.onSubmit}>
+          {(!isValid) && (
+            <Row>
+              <Col xs={{ size: 4, push: 4, pull: 4 }}>
+                <Alert color="danger" toggle={this.onDismiss} isOpen={this.state.alertVisible}>
+                  Data is not vlaid
+                </Alert>
+              </Col>
+            </Row>
+          )}
```
Comment (Mossie93) [inline src/components/users/LoginPage.js]: Clever...

File: src/index.js [hunk 2]
```diff
@@ -2,30 +2,43 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import configureStore from './store/configureStore';
 import { Provider } from 'react-redux';
-import { BrowserRouter as Router, Route } from 'react-router-dom'
+import {
+  BrowserRouter as Router,
+  Route,
+  Link,
+  Switch,
+} from 'react-router-dom'
```
Comment (Mossie93) [inline src/index.js]: missing semicolon

File: src/index.js [hunk 3]
```diff
@@ -2,30 +2,43 @@ import React from 'react';
...
 import FetchMembershipsForm from './components/memberships/FetchMembershipsForm';
-import { Link } from 'react-router-dom'
 import TeamsPage from './components/teams/TeamsPage';
 import TeamMembersPage from './components/team_members/TeamMembersPage';
 import MembershipsPage from './components/memberships/MembershipsPage';
+import LoginPage from './components/users/LoginPage';
 import 'bootstrap/dist/css/bootstrap.css';
 import './stylesheets/leader_app.css';
 import { Container } from 'reactstrap';
+import { PrivateRoute } from './helpers/routeHelper'
```
Comment (Mossie93) [inline src/index.js]: missing semicolon

File: src/api/apis.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import axios from 'axios'
```
Comment (Mossie93) [inline src/api/apis.js]: missing semicolon

File: src/api/authApi.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import { peopleApi } from './apis'
```
Comment (Mossie93) [inline src/api/authApi.js]: missing semicolon

File: src/api/membershipApi.js [hunk 6]
```diff
@@ -1,20 +1,9 @@
-import axios from 'axios'
+import { peopleApi } from './apis'
```
Comment (Mossie93) [inline src/api/membershipApi.js]: missing semicolon

**Review by Mossie93 — COMMENTED** [review]
Code logic looks good, style is almost ok too, but you forgot about some semicolons.

**Review by zuzannast — APPROVED** [review]

**Pull request merged**